generic_petg.xml.fdm_material

```diff
@@ -64,6 +64,28 @@ Generic PETG profile. The data in this file may not be correct for your specific
             </hotend>
         </machine>
 
+        <machine>
+            <machine_identifier manufacturer="Ultimaker B.V." product="Ultimaker 3"/>
+            <machine_identifier manufacturer="Ultimaker B.V." product="Ultimaker 3 Extended"/>
+            <setting key="heated bed temperature">85</setting>
+            <hotend id="BB 0.4" />
+            <hotend id="BB 0.8" />
+            <hotend id="AA 0.25">
+                <setting key="hardware compatible">yes</setting>
+                <setting key="print temperature">230</setting>
+                <setting key="standby temperature">100</setting>
+                <setting key="retraction amount">6.5</setting>
+            </hotend>
+            <hotend id="AA 0.4">
+                <setting key="hardware compatible">yes</setting>
+                <setting key="standby temperature">100</setting>
+                <setting key="print cooling">20</setting>
```

Comment:
This setting isn't used by the printer. Perhaps used by Cura? But, then, why only specified for the AA 0.4 nozzle?

Reply:
this is used by Cura, and is mixed between printcores. so didn't change things here for now.

Reply:
ok